src/js/facility-locator/reducers/searchQuery.js

```diff
@@ -0,0 +1,22 @@
+import { SEARCH_QUERY_UPDATED } from '../actions';
+// TODO (bshyong): flesh out shape of service Type object/options
+const INITIAL_STATE = {
+  searchString: '',
+  serviceType: null,
+  position: {
```

Comment:
I will create a Github issue with this and other similar decisions that will need to be made. On it.